src/main/java/core/basesyntax/io/FileReaderImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.io;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String fileName) {
+        List<String> info = new ArrayList<>();
+        try (BufferedReader bufferedReader = new BufferedReader(new java.io.FileReader(fileName))) {
+            String line;
+            while ((line = bufferedReader.readLine()) != null) {
+                info.add(line);
+            }
+
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t read file", e);
+        }
+        return info;
+    }
```

Comment:
According to the checklist, avoid using hardcoded values in your solution. It is better to define the file path as a parameter of the method and pass it when calling the method. This will make the code more flexible and easier to test.